backend/src/main/java/storage/PostgresSQLConnector.java

```diff
@@ -1,19 +1,22 @@
 package storage;
 
 
-
 import java.sql.*;
 import java.util.Properties;
 
 public class PostgresSQLConnector implements DBInterface {
 
-    private static final String CONNECTION_URL = "jdbc:postgresql://db:5432/svereadb";
+    private static final String CONNECTION_URL_DOCKER = "jdbc:postgresql://db:5432/svereadb";
+    private static final String CONNECTION_URL_DEBUG = "jdbc:postgresql://localhost:5432/svereadb";
     private static final String USERNAME = "admin";
     private static final String PWD = "1234";
+
+    private static String CONNECTION_URL;
     final Properties connectionProperties = new Properties();
     Connection conn = null;
 
-    public PostgresSQLConnector() {
+    public PostgresSQLConnector(boolean debug) {
+        CONNECTION_URL = debug ? CONNECTION_URL_DEBUG : CONNECTION_URL_DOCKER;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this assignment of "CONNECTION_URL". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3010)